src/pages/wallet-import-keystore/wallet-import-keystore.spec.ts

```diff
@@ -250,9 +250,9 @@ describe("WalletImportKeystorePage", () => {
   });
 
   it("should abort the import when the data is not valid", (done) => {
-    spyOn(comp, "dataIsValid").and.returnValue(false);
+    spyOn(comp, "clipboardDataIsValid").and.returnValue(false);
 
-    comp.import().then(
+    comp.importByClipboard().then(
```

Comment:
I don't understand how you are checking here if the import is aborted. I don't see any checks for this.

Reply:
Changed to promise.reject in the code and is properly called next request